app/models/description_change_validation_request.rb

```diff
@@ -12,6 +12,10 @@ class DescriptionChangeValidationRequest < ApplicationRecord
 
   validates :proposed_description, presence: true
   validate :rejected_reason_is_present?
+  validate :allows_only_one_open_description_change, on: :create
+  validate :planning_application_has_not_been_determined, on: :create
+
+  scope :open_change_created_over_5_days_ago, -> { where("state = 'open' AND created_at > ?", 5.business_days.ago) }
```

Comment:
you get a free `open` scope with AASM:

```ruby
scope :autoaccept, -> { open.where("created_at >= ? ", 5.business.days.ago) }
```

is it greater or greater or equal? always get confused about these things

Reply:
went with > to give it a little more time since the business day is defined as a 9 to 5 it can be a little strict - I'm not sure I understand this autoaccept scope comment ?